ion-schema/src/isl/isl_constraint.rs

```diff
@@ -483,7 +485,9 @@ pub(crate) enum IslConstraintImpl {
     Contains(Vec<Element>),
     ContentClosed,
     ContainerLength(Range),
-    Element(IslTypeRefImpl),
+    // For ISL 2.0 true/false is specified based on whether `distinct` annotation is present or not.
+    // Represents Element(type_reference, expected_distinct). None here is used for ISL 1.0 which doesn't support `distinct` elements.
+    Element(IslTypeRefImpl, Option<bool>),
```

Comment:
What's the reasoning for having `None` instead of just using `false` for ISL 1.0?